README.md

```diff
@@ -39,6 +39,15 @@ To print each json object to a new line, set `new_line` to `true`:
 #{formatter => {jsonformat, #{ new_line => true }}}
 ```
 
+To control what is being included in the log object from the metadata, there
+are two ways. One can opt-out from fields. Default opts out is `[report_cb]`.
+
+    #{ meta_without => [report_cb, gl, file, domain] }
```

Comment:
I think we need wild cards here.

At least "meta_without => all" or "meta_without => ['*'].